lib/cache.js

```diff
@@ -47,7 +47,7 @@ var jdgClient = function() {
   memcached = new Memcached(process.env.JDG_SERVICE_HOSTNAME + ':' + process.env.JDG_SERVICE_PORT, options);
 };
 
-var redisClient = function() {
+redisClient = function() {
```

Comment:
Would it be preferable here to instead remove the uninitialized `redisClient` from line 6, which would mean that the variable never has the undefined value? Visually this appears to be at the same scope level, and it's not (yet...thankfully) referenced before this.

Reply:
Done:-)